osu.Game.Rulesets.Catch/Mods/CatchModShrinking.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using osu.Framework.Audio.Track;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Beatmaps;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Catch.Mods
+{
+    public class CatchModShrinking : Mod, IUpdatableByPlayfield, IApplicableToBeatmap, IApplicableToTrack
+    {
+        public override string Name => "Shrinking Playfield";
+        public override string Acronym => "SP";
+        public override string Description => "The playfield will shrink over time.";
+        public override double ScoreMultiplier => 1;
+        public override IconUsage? Icon => FontAwesome.Solid.CompressArrowsAlt;
+
+        /// <summary>
+        /// The point in the beatmap at which the final ramping rate should be reached.
+        /// </summary>
```

Comment:
This can't be copied verbatim without even changing all of the references to "track"/"rate" to what is actually relevant to this mod (which is the playfield scale I guess).

Reply:
Ahh right, that's something I kept in while working on it to make things easier but forgot to edit afterwards. Will change it